Makefile

```diff
@@ -2,14 +2,21 @@
 build:
 	docker build -t local/labelbox-python:test .
 
+
+test-local: build
+	docker run -it -v ${PWD}:/usr/src -w /usr/src \
+		-e LABELBOX_TEST_ENVIRON="local" \
+		-e LABELBOX_TEST_API_KEY_LOCAL=${LABELBOX_TEST_API_KEY} \
+		local/labelbox-python:test pytest $(PATH_TO_TEST) -svvx
+
 test-staging: build
 	docker run -it -v ${PWD}:/usr/src -w /usr/src \
 		-e LABELBOX_TEST_ENVIRON="staging" \
-		-e LABELBOX_TEST_API_KEY_STAGING=${LABELBOX_TEST_API_KEY_STAGING} \
+		-e LABELBOX_TEST_API_KEY_STAGING=${LABELBOX_TEST_API_KEY} \
 		local/labelbox-python:test pytest $(PATH_TO_TEST) -svvx
 
 test-prod: build
 	docker run -it -v ${PWD}:/usr/src -w /usr/src \
 		-e LABELBOX_TEST_ENVIRON="prod" \
-		-e LABELBOX_TEST_API_KEY_PROD=${LABELBOX_TEST_API_KEY_PROD} \
+		-e LABELBOX_TEST_API_KEY_PROD=${LABELBOX_TEST_API_KEY} \
```

Comment:
These shouldn't all have the same key name.

Reply:
Oh, yeah, i forgot i made this chane. 
Can you explain why? the key changes like daily, so i feel like nobody is going to export all of them at once and then expect them to work later, since they just . So it kind of felt like to me it was more useful to have them all be the same name so you didnt need to rename the variable name when switching runs between local & staging. 

But i dont have a strong opinion on it and im more than happy to undo this.